src/utils/decorators/input/input.scss

```diff
@@ -255,6 +255,8 @@
   .common-input__input--#{$type},
   .common-input__input--#{$type}:hover {
     border-color: $color !important; // this is the correct use of !important
+    padding-right: 60px;
```

Comment:
Is this not going to affect ALL inputs? Whereas you want to target just dropdowns (or inputs with a button to the right?) Date for example may need updating in a different way.

Reply:
Our scope was just for dropdowns. We can test how it affects other inputs and see if there are any issues, or I can target the change just for dropdowns

Reply:
You need to regression test other inputs at least, as this is a shared module you have changed. I suspect they will be negatively affected in this case.